src/main/java/teammates/storage/api/FeedbackResponseCommentsDb.java

```diff
@@ -333,11 +335,12 @@ public void deleteCommentById(Long id) {
         ofy().delete().keys(getEntityQueryKeys(id)).now();
     }
 
-    private FeedbackResponseComment getFeedbackResponseCommentEntity(String courseId, Instant createdAt, String giverEmail) {
+    private FeedbackResponseComment getFeedbackResponseCommentEntity(String courseId, Instant createdAt,
+                                                                     String commentGiver) {
         return load()
                 .filter("courseId =", courseId)
                 .filter("createdAt =", TimeHelper.convertInstantToDate(createdAt))
-                .filter("giverEmail =", giverEmail)
+                .filter("commentGiver =", commentGiver)
```

Comment:
Please notice that some data still exists (which use the old `giverEmail`) in the DataStore. Therefore, you shouldn't change the query.

Reply:
Changing `giverEmail` to `commentGiver` wouldn't be part of data migration? I was thinking it might become confusing to have two different names for same attribute.

Reply:
Think about the existing comment. 
You will need to write a data migration script.
My suggestion would be to get this PR merged (to get the feature delivered in this GSoC). After that, if you have spare time, we can fix this naming convention.